scantailor.desktop

```
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Scan Tailor
+Comment=An interactive post-processing tool for scanned pages.
+Exec=scantailor %u
+Icon=scantailor
```

Should be this instead
```suggestion
Icon=org.scantailor.ScanTailor
```